storage/raftstorage.go

```diff
@@ -206,6 +166,13 @@ func (r *Range) FirstIndex() (uint64, error) {
 	return ts.Index + 1, nil
 }
 
+// loadAppliedIndex retrieves the applied index from storage.
+func (r *Range) loadAppliedIndex() (uint64, error) {
+	return loadAppliedIndex(r.rm.Engine(), r.Desc().RaftID)
+}
+
+// loadAppliedIndex retrieves the applied index for the given
+// raftID from the supplied engine.
 func loadAppliedIndex(eng engine.Engine, raftID int64) (uint64, error) {
```

Comment:
Why do we have loadAppliedIndex in both a static function and a method of `*Range`?